zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/DefaultExporterResourceProvider.java

```diff
@@ -81,42 +81,42 @@ public void init(final ExporterConfiguration configuration) {
     templateDescriptorsMap =
         Map.of(
             ListViewTemplate.class,
-            new ListViewTemplate(globalPrefix, isElasticsearch),
+            new ListViewTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             VariableTemplate.class,
-            new VariableTemplate(globalPrefix, isElasticsearch),
+            new VariableTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             PostImporterQueueTemplate.class,
-            new PostImporterQueueTemplate(globalPrefix, isElasticsearch),
+            new PostImporterQueueTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             FlowNodeInstanceTemplate.class,
-            new FlowNodeInstanceTemplate(globalPrefix, isElasticsearch),
+            new FlowNodeInstanceTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             IncidentTemplate.class,
-            new IncidentTemplate(globalPrefix, isElasticsearch),
+            new IncidentTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             SequenceFlowTemplate.class,
-            new SequenceFlowTemplate(globalPrefix, isElasticsearch),
+            new SequenceFlowTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             DecisionInstanceTemplate.class,
-            new DecisionInstanceTemplate(globalPrefix, isElasticsearch),
+            new DecisionInstanceTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             EventTemplate.class,
-            new EventTemplate(globalPrefix, isElasticsearch),
+            new EventTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true),
             TaskTemplate.class,
-            new TaskTemplate(globalPrefix, isElasticsearch));
+            new TaskTemplate(globalPrefix, isElasticsearch).withHasGlobalPrefix(true));
```

Comment:
🙃 Feels duplicated, as we specify the prefix in the constructor 🤔

Reply:
there's no way to know (to my knowledge) based on the value of the prefix whether we are using a global prefix or a local prefix (like in the operate module) I think this is the most succinct solution without adding a new constructor to dozens of classes.

Reply:
Lets sync